Build.mak

```diff
@@ -1,9 +1,12 @@
 # Treat deprecations as errors to ensure ocean doesn't use own deprecated
-# symbols internally
-ifeq ($(DVER),1)
-override DFLAGS := $(filter-out -di,$(DFLAGS)) -v2 -v2=-static-arr-params -v2=-volatile
-else
-override DFLAGS += -de
+# symbols internally. Disable it on explicit flag to make possible regression
+# testing in D upstream.
+ifndef ALLOW_DEPRECATIONS
```

Comment:
I won't say this is a blocker, but I would compare against `ALLOW_DEPRECATIONS=1` to avoid confusion if someone thinks it can explicitly disable deprecations by using `ALLOW_DEPRECATIONS=0`, which is a common practice in Makd.